bin/vcs-jump

```diff
@@ -152,6 +146,8 @@ if ARGV.count < 1
 end
 
 mode = ARGV.shift
+FileUtils.cd(ARGV.shift)
```

Comment:
This is where the conflict lies: you're shifting off the beginning of the array, but the argument you're trying to grab you actually added to the end of it; so if the command was something like `:VcsJump grep something`, you will invoke the binary with `grep something some/path` and here you'll shift off `something` and try to `cd` to that and probably fail.